about-me.html

```diff
@@ -0,0 +1,172 @@
+---
+layout: default
+title: About me
+---
+
+<h1>About me</h1>
+
+<section id="about-me">
+    <p>
+        My name is Kévin and I am a software engineer specialized in backend applications.
+        I started web development as an autodidact in 2008 and acquired a strong knowledge
+        and expertise in this field over time.
+    </p>
+
+    <p>
+        I want this page to be a "<em>not-so-formal</em>" description of who I am,
```

Comment:
Not really convinced about this part.

Reply:
What do you mean? You're not convinced that it's not formal? Or maybe you don't believe that it describes me accurately?